frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/patient-popup/patient-popup.component.ts

```diff
@@ -40,6 +47,25 @@ export class PatientPopupComponent implements PopupComponent, OnInit {
 
     ngOnInit(): void {
         this.patient$ = this.store.select(getSelectPatient(this.patientId));
+        this.patientStatus$ = this.store.select(
+            createSelector(
+                getSelectPatient(this.patientId),
+                selectPretriageFlag,
+                (patient, pretriageFlag) =>
+                    !pretriageFlag ||
+                    patient.treatmentTime >= this.secondsUntilRealStatus
+                        ? patient.realStatus
+                        : patient.visibleStatus
```

Comment:
This should be a static function in Patient

Reply:
The function could be named `getVisibleStatus()` and the property `visibleStatus` should be renamed into `pretriageStatus`